gsr_booking/booking_logic.py

```diff
@@ -0,0 +1,255 @@
+import requests
+import datetime
+import math
+import random
+
+
+class BookingLogic:
+
+    DATE_FORMAT_STR = "%Y-%m-%dT%H:%M:%S%z"
```

Comment:
I think this is just the ISO datetime format? if so i think you can use this function: https://docs.python.org/3/library/datetime.html#datetime.datetime.isoformat